### ISSUE TITLE
MRE's nested containers slow down game when many containers are present in one map chunk.

### ISSUE THREAD

**Issue by freijie** [CleverRaven/Cataclysm-DDA#50932]
### Describe the bug

Noticed significant slow down when looting piles(as in more than a dozen or two) of MREs. 
To full extent slow down was visible during looting of Military Base location. Approaching map chunk storage with MRE resulted in:
- freeze upon entering the chunk, measuring some 5-10 seconds
- jump of fast forwarding time during craft and wait by a couple of orders of magnitude, from 10 seconds for half-day ingame to 10 real life seconds for several minutes of ingame time.

Burning MRE shelves alone solves this issue, no freezes upon entering map chunk,  time fast forwards at reasonable speed, so slowdown is pretty much due to MRE packages alone - there is a huge number of other items in base.

### Steps To Reproduce

Get a lot of MREs in one area, enjoy the lag while game processes them.

### Expected behavior

I'd expect MRE package to return to non-nested nature to:
Improve speed to huge extent
Reduce clutter and QoL issue where you have to open many game windows just to finish one MRE meal, for negligible gameplay benefits of close to real representation and separate seals for packages.

As a bonus, ability to batch-disassembling of MRE would be really, really nice for gameplay/QoL for cases like military base or looting large soldier groups.

### Versions and configuration

- OS: Windows 10
- Game Version: 8b56579
- Graphics version: Tiles
- Ingame language: English
- Mods loaded: dda, np_npc_food, package_bionic_professions, aftershock, aftershock_exoplanet, blazeindustries, MMA, No_Rail_Stations, Graphical_Overmap, No_Fungi, speedydex, stats_through_kills, StatsThroughSkills

### Additional context

[Burzum.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/7023867/Burzum.zip)
Savegame for illtustration, notice that its an Aftershock Exoplanet world.

**Comment by hexagonrecursion** [comment]
I can confirm that the game does get noticeably faster after I burn the MREs. It is still a bit sluggish though
- OS: Linux
    - OS Version: LSB Version: :core-4.1-amd64:core-4.1-noarch; Distributor ID: Fedora; Description: Fedora release 33 (Thirty Three); Release: 33; Codename: ThirtyThree; 
- Game Version: 8051d7d [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions]
]

**Comment by YourCompanionCube** [comment]
i would like to add that the old version let me autopickup only the good MREs where under the current system i can only pick up the packages they come in including the empty ones

**Comment by RandDenner** [comment]
I just tested this in 0.F-1.
In a military warehouse chock full of MREs and IFAKs, crafting a simple set of storage shelves took at least 10x-20x longer than in a nearby map away from the items.
Additionally noticed the map save lag mentioned as well.
Can provide a save if requested.

**Comment by YourCompanionCube** [comment]
> These MREs (& IFAKs) should just be single items until opened (by activating?).
> Interaction by the player to cause the contents to generate would solve the issues

i agree with this, it would still allow for military zombies to have partial MREs but massively reduce the lag and clutter of full ones
